packages/desktop-gui/cypress/integration/setup_project_modal_spec.js

```diff
@@ -447,13 +447,20 @@ describe('Set Up Project', function () {
       cy.get('.modal').contains('Log In to Dashboard')
     })
 
+    it('closes login modal', () => {
+      cy.get('.modal').contains('Log In to Dashboard')
+      cy.get('.close').click()
+      cy.get('.btn').contains('Set up project').click()
```

Comment:
we don't have good selectors there?

Reply:
not really, i think this is based off the other tests in this file